src/pages/VideoContentPage/VideoContentPage.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
add _redirects file for netlify redirects fix.

Reply:
ok added redirects